src/spyglass/common/common_ephys.py

```diff
@@ -88,7 +88,7 @@ class Electrode(SpyglassMixin, dj.Imported):
     x = NULL: float                         # the x coordinate of the electrode position in the brain
     y = NULL: float                         # the y coordinate of the electrode position in the brain
     z = NULL: float                         # the z coordinate of the electrode position in the brain
-    filtering: varchar(2000)                # description of the signal filtering
+    filtering: = "unfiltered": varchar(2000)      # description of the signal filtering
```

Comment:
This is a table edit that would need to be reflected in changelog/release notes - I added some notes on this to the PR template: 

```python
from spyglass.common.common_ephys import Electrode

Electrode.alter()
```

Because this is adding a default value to a secondary key, there won't be any issues running the alter, but it's worth noting for the unlikely case in which someone is relying on a DJ err for a data w/o fitering

Reply:
No need anymore